todo_app/data/mongodb.py

```diff
@@ -34,8 +35,43 @@ def initdb(cls):
         print(f"Connected to {server}/{database}")
         cls.database=cls.client[database]
         cls.items_collection = cls.database.items_collection
+        cls.user_collection = cls.database.users
         cls.init=True
 
+    @classmethod
+    def get_user(cls, username):
```

Comment:
As above, this function says it expects a username but in practice is handed a user id